### PR TITLE
Update j/input,j/session,phar-stream-wrapper. Pinned sodium_compat

### PR DESCRIPTION
**Pull request by HLeithner**
Pinned sodium_compat because of file case change, atm this could
not be handled by the Joomla! updater.

Replace PR #25279

Pull Request for Issue #25277

Maybe @jurihahn and @LoRexxar can test the phar-stream-wrapper update?.

Sodium_compat is now pinned to version 1.9.1 till we have a solution for the filename with the wrong case.

@mbabker can you check this PR?

### CODE REVIEW

File: libraries/vendor/composer/ClassLoader.php [hunk 1]
```diff
@@ -279,7 +279,7 @@ public function isClassMapAuthoritative()
      */
     public function setApcuPrefix($apcuPrefix)
     {
-        $this->apcuPrefix = function_exists('apcu_fetch') && filter_var(ini_get('apc.enabled'), FILTER_VALIDATE_BOOLEAN) ? $apcuPrefix : null;
```
Comment (wilsonge) [inline libraries/vendor/composer/ClassLoader.php]: Looks like you've used an old version of composer here. you're reverting "correct" code

Comment (HLeithner) [reply libraries/vendor/composer/ClassLoader.php]: rebuild with the latest and greatest done.

**Review by wilsonge — APPROVED** [review]
Looks good on review

**Comment by rdeutz** [comment]
I looks good to me too, I didn only a code review. Sorry don't have time for a real test

**Comment by HLeithner** [comment]
thx

**Comment by wilsonge** [comment]
Did a real test just now. Nothing was obviously broken doing a quick browse around the frontend/backend

**Pull request merged**